en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md

```diff
@@ -46,13 +46,13 @@ for more information on what you can monitor using EI Analytics.
 
 Let's start the servers in the given order.
 
-### Starting the Analytics Worker
+### Starting the Analytics Server
 
 !!! Note
     Be sure to start the **Analytics** server before [starting the Micro Integrator](#starting-the-micro-integrator).
 
-1.  Open a terminal and navigate to the `<EI_6.x.x_HOME>/wso2/analytics/bin` directory.
-2.  Start the Analytics Worker runtime by executing the following command:
+1.  Open a terminal and navigate to the `<EI_ANALYTICS_HOME>/wso2/analytics/bin` directory.
```

Comment:
The path here should change as well. `<EI_ANALYTICS_HOME>/bin`

Reply:
fixed